Source/WebCore/css/StyleProperties.cpp

```diff
@@ -705,16 +705,17 @@ String StyleProperties::get4Values(const StylePropertyShorthand& shorthand) cons
     if (!top.value() || !right.value() || !bottom.value() || !left.value())
         return String();
 
+    if (top.isImportant() != right.isImportant() || right.isImportant() != bottom.isImportant() || bottom.isImportant() != left.isImportant())
+        return String();
+
```

Comment:
This is unnecessary, already checked in `StyleProperties::commonShorthandChecks`.